src/store/eventing/active.ts

```diff
@@ -1,14 +1,16 @@
 import eventManager from '../../common/EventManager';
-import { AppState } from '../types';
 import { getActiveAnnotationId } from '../annotations';
+import { getFileVersionId } from '../options';
+import { AppState } from '../types';
 import { Event } from '../../@types';
 
 const handleActiveAnnotationEvents = (prevState: AppState, nextState: AppState): void => {
     const prevActiveAnnotationId = getActiveAnnotationId(prevState);
     const nextActiveAnnotationId = getActiveAnnotationId(nextState);
+    const fileVersionId = getFileVersionId(nextState);
 
     if (prevActiveAnnotationId !== nextActiveAnnotationId) {
-        eventManager.emit(Event.ACTIVE_CHANGE, nextActiveAnnotationId);
+        eventManager.emit(Event.ACTIVE_CHANGE, { annotationId: nextActiveAnnotationId, fileVersionId });
```

Comment:
Looks like a breaking change. Do you plan to merge/release both this and BUIE at the same time?

Reply:
Yeah, will need to bump both at the same time